src/scripts/components/Play.js

```diff
@@ -319,37 +319,37 @@ class Play extends Component<Props, State> {
   facebook () {
     if (this.props.video) {
       var baseurl = 'https://www.facebook.com/sharer/sharer.php?u='
-      return baseurl + this.portalUrl() + '/embed/' + this.props.video.id
+      return baseurl + this.portalUrl() + '/play/' + this.props.video.id
     }
   }
   twitter () {
     if (this.props.video) {
       var baseurl = 'https://twitter.com/intent/tweet'
-      var url = '?url=' + this.portalUrl() + '/embed/' + this.props.video.id
+      var url = '?url=' + this.portalUrl() + '/play/' + this.props.video.id
       var text = '&text=🎬 Worth a watch: ' + this.props.video.title
       return baseurl + url + text
     }
   }
   whatsapp () {
```

Comment:
Can you rename all of these methods to be more clear? I think a better name could be `getWhatsAppUrl`